<launch>
  <arg name="device_ids" default="usb://*;tcp://piksi:55555"/> <!-- E.g. "usb://*;tcp://192.168.222.0:55555;serial:///dev/ttyS5@115200" -->
  <node name="piksi_multi_cpp_base" pkg="piksi_multi_cpp" type="piksi_multi" output="log" required="true">
      <param name="device_ids" value="$(arg device_ids)" />
      <param name="use_gps_time" value="true" />
      <param name="autostart_base_sampling" value="false" />
      <param name="num_desired_fixes" value="2000" />
      <param name="log_observations_to_file" value="false" />
      <!--param name="log_dir" value="/tmp" /-->
      <!--param name="observation_filename" value="raw_observations" /-->

      <param name="base_station_receiver_0/udp_port"
             value="26078"/> <!-- Port on which UDP broadcasts are sent -->

      <param name="base_station_receiver_0/udp_broadcast_interfaces"
             value="lo;enp1s0;enp4s0;eth0"/> <!-- Semicolon separate list of interfaces on which observations should be sent via network broadcast -->

      <param name="base_station_receiver_0/udp_unicast_targets"
             value=""/> <!-- Additional hosts to which observations are sent via unicast -->
  </node>
</launch>
